tests/c_languages/CPPCleanBearTest.py

```diff
@@ -17,6 +19,44 @@
 """
 
 
+test_include_paths_file = """
+#include "foo.h"
+#include "bar.h"
+
+int main() {
+    foo();
+    bar();
+    return 0;
+}
+"""
+
+
+def get_testdirectory_name(test_directory):
+    return os.path.join(os.path.dirname(__file__),
+                        'test_files',
+                        test_directory)
+
+
 CPPCleanBearTest = verify_local_bear(CPPCleanBear,
                                      valid_files=(good_file,),
-                                     invalid_files=(bad_file,))
+                                     invalid_files=(bad_file,
+                                                    test_include_paths_file,),
+                                     tempfile_kwargs={'suffix': '.cpp'})
+
+CPPCleanBearValidIncludeMultiplePathTest = verify_local_bear(
+    CPPCleanBear,
+    valid_files=(test_include_paths_file,),
+    invalid_files=(),
+    settings={'include_paths': escape(
+                get_testdirectory_name('headers1'), '\\') + ',' +
+              escape(
+                get_testdirectory_name('headers2'), '\\')},
```

Comment:
4 spaces indent relative to opening expression:
```
escape(
    get_testdirectory_name(...), ...)  # <-- note: 4 spaces indent at start of line
```
same above
(I would maybe reformat to
```
    settings={'include_paths':
                  escape(get_testdirectory_name('headers1'), '\\') + ',' +
                  escape(get_testdirectory_name('headers2'), '\\')},
```
)

Reply:
@Makman2 So there is some PEP8 issue with that 4 spaces indent. Its forcing me to do something like this : 
'''
settings={'include_paths':
                 escape(get_testdirectory_name('headers1'), '\\') + ',' +
                 escape(get_testdirectory_name('headers2'), '\\')},
'''

Reply:
@Makman2 done:)